pages/osx/cot.md

```diff
@@ -0,0 +1,24 @@
+# cot
+
+> The Plain-Text Editor for macOS.
+> More information: <https://coteditor.com/>.
+
+- Start CotEditor:
+
+`cot`
+
+- Open a new blank document:
+
+`cot --new`
+
+- Open specific files:
+
+`cot {{path/to/file1 path/to/file2 ...}}`
+
+- Open and wait until finished editing a specific file:
```

Comment:
```suggestion
- Open a specific file in the foreground (blocking the terminal until the editor is closed):
```

Reply:
This line is worded the same way that #8206 was worded. In addition to making this change, should I also open a PR to update the wording for `mate`?